db/migrations/create_chitter.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, peep VARCHAR(140));
```

Comment:
- Instructions in migrations don't have timestamp column (the page throws an error about that)